bin/improver-generate-topography-bands-mask

```diff
@@ -101,16 +104,27 @@ def main():
 
     if not os.path.exists(args.output_filepath) or args.force:
         orography = load_cube(args.input_filepath_standard_orography)
-        try:
-            landmask = load_cube(args.input_filepath_landmask)
-        except IOError as err:
-            msg = ("Loading land mask has been unsuccessful: {}. "
-                   "This may be because the land mask could not be "
-                   "located in {}; run "
-                   'improver-generate-landmask-ancillary first.').format(
-                       err, args.input_filepath_landmask)
+
+        landmask = None
+        if args.input_filepath_landmask:
+            try:
+                landmask = load_cube(args.input_filepath_landmask)
+            except IOError as err:
+                msg = ("Loading land mask has been unsuccessful: {}. "
+                       "This may be because the land mask could not be "
+                       "located in {}; run "
+                       'improver-generate-landmask-ancillary first.').format(
+                           err, args.input_filepath_landmask)
+            if thresholds_dict == THRESHOLDS_DICT:
+                thresholds_dict['land'] = thresholds_dict.pop(
```

Comment:
A comment would be helpful here explaining why we need to rename the dictionary item.  

Even better would be to pass down the logic from this interface into the plugin - so have the sea mask generated based on whether "landmask" exists rather than the name of the threshold key.  I see that an error is already raised in the CLI if the landsea mask can't be read.  However this might be a bit of work in GenerateOrographyBandAncils.gen_orography_masks() so I'm open to discussion.

Reply:
I think we still want it to be based on the key as someone might lazily copy the normal suite behaviour, that provides a landmask, but have written a custom thresholds json file using any_surface_type. In this case the landmask presence or absence is not really important. Raising an error if it is needed but absent remains the only vital test I think.